src/main/java/org/zalando/jackson/module/unknownproperty/UnknownPropertyModule.java

```diff
@@ -5,28 +5,44 @@
 import com.fasterxml.jackson.databind.Module;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
 
 public final class UnknownPropertyModule extends Module {
 
     private static final Logger LOG = LoggerFactory.getLogger(UnknownPropertyModule.class);
+    private static final Level LOG_LEVEL = Level.TRACE;
 
     private final Logger logger;
+    private final Level logLevel;
     private final String format;
 
     public UnknownPropertyModule() {
         this(LOG);
     }
 
     public UnknownPropertyModule(final Logger logger) {
-        this(logger, "Unknown property in {}: {}");
+        this(logger, LOG_LEVEL);
     }
-    
+
+    public UnknownPropertyModule(final Level logLevel) {
+        this(LOG, logLevel);
+    }
+
     public UnknownPropertyModule(final String format) {
-        this(LOG, format);
+        this(LOG, LOG_LEVEL, format);
+    }
+
+    public UnknownPropertyModule(final Logger logger, final Level logLevel) {
+        this(logger, logLevel, "Unknown property in {}: {}");
     }
 
     public UnknownPropertyModule(final Logger logger, final String format) {
+        this(logger, LOG_LEVEL, format);
+    }
+
+    public UnknownPropertyModule(final Logger logger, final Level logLevel, final String format) {
```

Comment:
How about keeping the constructors that we had before and introduce a `withLogLevel(Level)` method that usages look like this:

```java
new UnknownPropertyModule(logger).withLogLevel(DEBUG);
```

That way we can keep the number of constructors low and reduce complexity.

The withLogLevel method should then just create a new, slightly modified module:

```java
public UnknownPropertyModule withLogLevel(final Level level) {
    return new UnknownPropertyModule(logger, level, format);
}
```

Reply:
Also helps with coverage, since there is less new code to cover.